### ISSUE TITLE
STEM Accessibility: No space between "You answered yes/no to" and "Are enrolled in a STEM..."

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#6290]
When using a screen reader on the STEM eligibility page of the STEM form, the words "to" from the icon and "are" from the `Are enrolled in a STEM undergraduate degree program, or have earned a STEM degree and are now pursuing a teaching certification` requirement are not read as separate words. When using voiceover, it is apparent that the words are not separated by a space. These words should be separated so that users using screen readers can clearly understand their eligibility.

![Screen Shot 2020-02-28 at 9.26.02 AM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/ea3feddd-c189-4009-9dd6-faf8a4d99442)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/11783

**Comment by delli-gatti-michael** [comment]
CHanges are now present in Staging. Closing this issue
